structs.go

```diff
@@ -85,6 +85,9 @@ type Session struct {
 	// Stores a mapping of guild id's to VoiceConnections
 	VoiceConnections map[string]*VoiceConnection
 
+	// Switch to enable processing of voice related messages
+	DisableVoiceHandlers bool
```

Comment:
We usually do true by default and optionally disabled options
```suggestion
	// Whether to handle voice related gateway events
	HandleVoiceGateway bool
```